dist/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
- [ ] I think it will be better to ignore the ./dist folder too. Since it can be generated on the fly using webpack.